src/vivarium/types.py

```diff
@@ -17,7 +16,7 @@
 ClockTime = Time | int
 ClockStepSize = Timedelta | int
 
-ScalarValue = Number | Timedelta | Time
+ScalarValue = float | Timedelta | Time
```

Comment:
Shouldn't this be float or int?

Reply:
Good catch

Reply:
in typehinting, 'int' counts as float but not in isinstance() https://peps.python.org/pep-0484/#the-numeric-tower

Reply:
but I think it's important to relay to us that it can also be an int